README.md

```diff
@@ -22,13 +22,12 @@ To help merchants get started, we've put together a few notes for common environ
 
 Java version | TLSv1.2 support
 --- | ---
-6 and earlier | No support. A runtime update is required. (Except possibly for IBM Java. See note below.)
+5 and earlier | No support.
+6 | Available. TLSv1.2 must be explicitly enabled. Requires at least [Oracle Java version `6u115`](http://www.oracle.com/technetwork/java/javase/documentation/overview-156328.html) or [IBM V6 service refresh 10](http://www-01.ibm.com/support/knowledgecenter/SSYKE2_6.0.0/com.ibm.java.security.component.60.doc/security-component/jsse2Docs/overrideSSLprotocol.html).  A [PayPal SDK update](#paypal-java-sdk-support) or code change may be required.
```

Comment:
How about adding the patch version in addition to `6u115`? Here is [the direct link](http://www.oracle.com/technetwork/java/javase/documentation/overview-156328.html#6u115-b32) to the change log.